src/dbg/_exports.cpp

```diff
@@ -337,103 +337,104 @@ extern "C" DLL_EXPORT bool _dbg_addrinfoget(duint addr, SEGMENTREG segment, BRID
             Zydis cp;
             auto getregs = !bOnlyCipAutoComments || addr == titcontext.cip;
             disasmget(cp, addr, &instr, getregs);
-
-            //Ignore register values when not on CIP and OnlyCipAutoComments is enabled: https://github.com/x64dbg/x64dbg/issues/1383
-            if(!getregs)
-            {
-                for(int i = 0; i < instr.argcount; i++)
-                    instr.arg[i].value = instr.arg[i].constant;
-            }
-
-            for(int i = 0; i < instr.argcount; i++)
+            if(!cp.IsNop())

```

Comment:
The diff is extremely unclear, this is the only link you added right?

Reply:
Try this:
https://github.com/x64dbg/x64dbg/pull/1780/files?w=1